src/CategoricalArrays.jl

```diff
@@ -13,11 +13,14 @@ module CategoricalArrays
 
     using Compat
     using Reexport
+
+    # TODO: cannot @reexport in conditional, the below should be removed when 0.6 is deprecated
```

Comment:
Even inside `@static if VERSION > ...`?

Reply:
hm, don't know, will experiment

Reply:
apparently not.  I think what's happening is that when `@reexport` macro gets called at compile time, it checks for `Missings` and it does this regardless of whether it's in a conditional or not.  let me know if you think of an alternative...

Reply:
Concretely, it is a problem to reexport Missings.jl on 0.7? It should work fine there, and we can keep it that way until Femtocleaner removes the block.

Reply:
Well, I'm stumped.  It doesn't seem to work in the conditional statement no matter what.  Doesn't matter if `Missing`s is installed, doesn't even matter if you already have it imported into the context where `CategoricalArrays` is imported.  As far as I can tell right now there is aboslutely no way to get this inside a conditional statement.

Reply:
Let's make it unconditional for now, that's not a big deal since AFAIK all exported objects are also in Base now.